### PR TITLE
add:additional categories(scraper)

### PR DESCRIPTION
**Pull request by ARAVINT-S-A**
Added more categories to scrape relevant data

### CODE REVIEW

File: news_scraper/spiders/businessstandard.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ class BusinessStandardSpider(SitemapIndexSpider):
         "month": lambda d: d.strftime("%B").lower(),
     }
 
-    sitemap_rules = [(r"/markets/", "parse")]
+    sitemap_rules = [(r"/markets/", "parse"),(r"article/markets/", "parse"),(r"article/companies/", "parse"),(r"article/economy-policy/", "parse")]
```
Comment (darwin403) [inline news_scraper/spiders/businessstandard.py]: @ARAVINT-S-A  Are you sure it's `article/markets/` and not just `/markets/` along with the others? These rules need to be updated

File: news_scraper/spiders/businesstoday.py [hunk 2]
```diff
@@ -11,7 +11,7 @@ class BusinessTodaySpider(SitemapIndexSpider):
         "https://www.businesstoday.in/rssfeeds/date-wise-story-sitemap.xml?yyyy={year}&mm={month}&dd={day}",
     ]
 
-    sitemap_rules = [(r"/markets/", "parse")]
+    sitemap_rules = [(r"/markets/", "parse"),(r"/industry/", "parse"),(r"/india-at-100/", "parse")]
```
Comment (darwin403) [inline news_scraper/spiders/businesstoday.py]: @ARAVINT-S-A  `/india-at-100` doesnt seem to be a category, more like a general event and some articles under it.

File: news_scraper/spiders/outlookindia.py [hunk 3]
```diff
@@ -13,7 +13,7 @@ class OutlookIndiaSpider(SitemapIndexSpider):
         "https://www.outlookbusiness.com/sitemap/sitemap-daily-{year}-{month}-{day}.xml"
     ]
 
-    sitemap_rules = [(r"/markets/", "parse")]
+    sitemap_rules = [(r"/markets/", "parse"),(r"/corporate/", "parse"),(r"/budget/", "parse")]
```
Comment (darwin403) [inline news_scraper/spiders/outlookindia.py]: `/budget/` is a singular event. This can be removed @ARAVINT-S-A

File: tests/test_spiders.py [hunk 4]
```diff
@@ -129,6 +129,7 @@ def test_spider_crawl(spider: Spider):
         "https://www.thehindu.com/business/markets/markets-decline-in-early-trade/article68380310.ece",
         "https://www.thehindubusinessline.com/markets/stock-markets/brokerage-views-on-dabur-emkay-global-and-dart-insights/article68380231.ece",
         "https://zeenews.india.com/markets/pharma-healthcare-stocks-top-sectoral-gainers-in-trade-2716060.html",
+
```
Comment (darwin403) [inline tests/test_spiders.py]: @ARAVINT-S-A  Please add tests for checking if parsing an article from all the above mentioned new categories passes.

File: news_scraper/spiders/businessstandard.py (hunk 1 above)
Comment (ARAVINT-S-A) [reply news_scraper/spiders/businessstandard.py]: https://www.business-standard.com/article/markets/sebi-slaps-rs-51-lakh-penalties-on-18-entities-in-atl-case-122033101539_1.html

I saw few pages like this so I included that

**Review by darwin403 — CHANGES_REQUESTED** [review]
Hey, I've proposed some changes. Please correct them!

File: news_scraper/spiders/businesstoday.py (hunk 2 above)
Comment (ARAVINT-S-A) [reply news_scraper/spiders/businesstoday.py]: Yes its an event (economy summit) I saw few company names and news so I thought of including it.

File: news_scraper/spiders/outlookindia.py (hunk 3 above)
Comment (ARAVINT-S-A) [reply news_scraper/spiders/outlookindia.py]: Ok fine.

File: news_scraper/spiders/businessstandard.py (hunk 1 above)
Comment (darwin403) [reply news_scraper/spiders/businessstandard.py]: `/markets/` and `article/markets` are redundant. Unless something like `/something-else/markets` kind of urls also exist, this doesnt make sense. 

Either remove `/markets/` and keep all other categories with `/article/` like `/article/news`or drop `/article` and keep everything like `/news/`.

The trailing slash is also important `/article/markets/` is not the same as `article/markets`, a bit paedantic but better to be safe.

File: news_scraper/spiders/businesstoday.py (hunk 2 above)
Comment (darwin403) [reply news_scraper/spiders/businesstoday.py]: Okay, drop this for now. Way too specific.

File: tests/test_spiders.py (hunk 4 above)
Comment (darwin403) [reply tests/test_spiders.py]: This is to ensure that all the url with the above categories are indeed news articles in expected HTML format and not some other dynamic pages.

File: news_scraper/spiders/businessstandard.py [hunk 5]
```diff
@@ -16,7 +16,7 @@ class BusinessStandardSpider(SitemapIndexSpider):
         "month": lambda d: d.strftime("%B").lower(),
     }
 
-    sitemap_rules = [(r"/markets/", "parse")]
+    sitemap_rules = [(r"/markets/", "parse"),(r"article/companies/", "parse"),(r"article/economy-policy/", "parse")]
```
Comment (darwin403) [inline news_scraper/spiders/businessstandard.py]: @ARAVINT-S-A

```python
sitemap_rules = [(r"/markets/", "parse"),(r"/companies/", "parse"),(r"/economy-policy/", "parse")]

# OR

sitemap_rules = [(r"/article/markets/", "parse"),(r"/article/companies/", "parse"),(r"/article/economy-policy/", "parse")]
```

I think you misunderstood what I meant last time. Please change this to the above.

**Pull request merged**